### PR TITLE
upgrade to 0.2.5

### PR DESCRIPTION
**Pull request by double-y**
Change names of functions

Test
* devtools:test()
* use functions installing this to the libpath in .exploratory dir

### CODE REVIEW

File: R/model_builder.R [hunk 1]
```diff
@@ -3,18 +3,29 @@
 
 #' Create do wrapper function with source data
 #' @return do wrapper function
-do_data <- function(funcname) {
-  ret <- function(df, ..., keep.source = FALSE){
+build_data <- function(funcname) {
+  ret <- function(df, ..., keep.source = FALSE, augment=F){
```
Comment (hideaki) [inline R/model_builder.R]: Isn't FALSE and F same thing? Should we uniformly use only one of those?

File: R/model_builder.R [hunk 2]
```diff
@@ -23,78 +34,144 @@ do_data <- function(funcname) {
...
+      %>%  dplyr::do_(.dots=setNames(list(~build_kmeans_each(.), ~(.)), c(model_column, source_column)))
+    )
+    # Add a class for Exploratyry to recognize the type of .source.data
+    class(output[[source_column]]) <- c("list", ".source.data")
+  } else {
+    output <- (
+      df
+      %>%  dplyr::do_(.dots=setNames(list(~build_kmeans_each(.)), model_column))
+    )
+    if(augment){
```
Comment (hideaki) [inline R/model_builder.R]: Why is this if(augment) case only in under else and not under if(keep.source)?
I guess keep.source implies augment at the next step? Anyway some comment would help.

Comment (double-y) [inline R/model_builder.R]: Yes, augment should be out from there.
I'll change the first condition to `keep.source | !augment` and put out the augment condition from the else block

File: R/model_builder.R (hunk 1 above)
Comment (double-y) [inline R/model_builder.R]: Yes, I'll change them to FALSE

**Pull request merged**